Library is not compiling

The library is not compiling due to narrowing errors

```
PATH/Grafici-GFX/src/Boundary.h: In member function 'virtual CartesianVector<float> PolarBoundary::project(CartesianVector<float>) const':
PATH/Grafici-GFX/src/Boundary.h:222:49: error: narrowing conversion of '(((double)((const PolarBoundary*)this)->PolarBoundary::<anonymous>.Boundary::_x.Range<T>::mid<float>()) + (((double)((((const PolarBoundary*)this)->PolarBoundary::<anonymous>.Boundary::_x.Range<T>::delta<float>() * 5.0e-1f) * radius)) * cos((((double)(angle * 2.0e+0)) * 3.1415926535897931e+0))))' from 'double' to 'float' inside { } [-Werror=narrowing]
   return CartesianVector<DisplayNorm>{ _x.mid() + (radius * (_x.delta() / 2)) * cos(angle * 2 * M_PI),
                                                 ^
PATH/Grafici-GFX/src/Boundary.h:223:46: error: narrowing conversion of '(((double)((const PolarBoundary*)this)->PolarBoundary::<anonymous>.Boundary::_y.Range<T>::mid<float>()) + (((double)((((const PolarBoundary*)this)->PolarBoundary::<anonymous>.Boundary::_y.Range<T>::delta<float>() * 5.0e-1f) * radius)) * sin((((double)(angle * 2.0e+0)) * 3.1415926535897931e+0))))' from 'double' to 'float' inside { } [-Werror=narrowing]
                                     _y.mid() + (radius * (_y.delta() / 2)) * sin(angle * 2 * M_PI) };
                                              ^
In file included from PATH/Grafici-GFX/src/Grafici.h:35:0,
                 from /home/walid/org/projects/development/02_internal/arduino/projects/room_logger_epaper/room_logger_epaper.ino:67:
PATH/Grafici-GFX/src/Plotter/Bubblemap.h: In member function 'virtual void Plotter::Bubblemap::draw_block(const Display&, const Boundary&, const ColorMap&, Range<float>, Range<float>, DataNorm) const':
PATH/Grafici-GFX/src/Plotter/Bubblemap.h:19:98: error: narrowing conversion of '(((double)(x.Range<T>::delta<float>() * c)) * 5.0e-1)' from 'double' to 'float' inside { } [-Werror=narrowing]
   display.fillCircle(boundary.project({ x.mid(), y.mid() }), boundary.projectAbs({ c * x.delta() / 2.0, c * y.delta() / 2.0 }), colorMap.project(c));
                                                                                                  ^
PATH/Grafici-GFX/src/Plotter/Bubblemap.h:19:119: error: narrowing conversion of '(((double)(y.Range<T>::delta<float>() * c)) * 5.0e-1)' from 'double' to 'float' inside { } [-Werror=narrowing]
   display.fillCircle(boundary.project({ x.mid(), y.mid() }), boundary.projectAbs({ c * x.delta() / 2.0, c * y.delta() / 2.0 }), colorMap.project(c));
```

--- a/src/Boundary.h
+++ b/src/Boundary.h
@@ -219,8 +219,11 @@ class PolarBoundary : public Boundary
 	{
 		DisplayNorm angle = a.map(data.x());
 		DisplayNorm radius = r.map(data.y());
-		return CartesianVector<DisplayNorm>{ _x.mid() + (radius * (_x.delta() / 2)) * cos(angle * 2 * M_PI),
-			                                 _y.mid() + (radius * (_y.delta() / 2)) * sin(angle * 2 * M_PI) };
+
+		DisplayNorm x = _x.mid() + (radius * (_x.delta() / 2)) * cos(angle * 2 * M_PI);
+		DisplayNorm y = _y.mid() + (radius * (_y.delta() / 2)) * sin(angle * 2 * M_PI);
+
+		return CartesianVector<DisplayNorm>{ x, y};
 	};
 
   private:

--- a/src/Plotter/Bubblemap.h
+++ b/src/Plotter/Bubblemap.h
@@ -16,7 +16,10 @@ class Bubblemap : public Heatmap
 	                Range<DataNorm> y,
 	                DataNorm c) const override
 	{
-		display.fillCircle(boundary.project({ x.mid(), y.mid() }), boundary.projectAbs({ c * x.delta() / 2.0, c * y.delta() / 2.0 }), colorMap.project(c));
+		DataNorm u = c * x.delta() / 2.0;
+		DataNorm v = c * y.delta() / 2.0;
+
+		display.fillCircle(boundary.project({ x.mid(), y.mid() }), boundary.projectAbs({ u, v }), colorMap.project(c));
 	};
 };
 

